docker/ledger-api-testtool.docker

```diff
@@ -13,8 +13,8 @@
 # limitations under the License.
 FROM ubuntu:bionic
 
-ARG DAML_SDK_VERSION=0.13.21
-ARG DAML_REPO_VERSION=100.13.21
+ARG DAML_SDK_VERSION=0.13.37
+ARG DAML_REPO_VERSION=100.13.36
```

Comment:
@scealiontach shouldn't these both be 37?

Reply:
path to test tool in bintray changed slightly : new path is
`https://bintray.com/api/v1/content/digitalassetsdk/DigitalAssetSDK/com/daml/ledger/testtool/ledger-api-test-tool/${sdkVersion}/ledger-api-test-tool-${sdkVersion}.jar?bt_package=sdk-components`

Reply:
resolved